src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/GoToDefinitionTests.cs

```diff
@@ -62,8 +62,8 @@ public async Task GoToDefinition_ComponentAttribute()
             var version = await TestServices.Shell.GetVersionAsync(HangMitigatingCancellationToken);
             if (version < new System.Version(17, 1, 32113, 165))
```

Comment:
@davidwengier can this be re-enabled?

Reply:
I guess so.. comment it out and see if the tests pass :D